### PR TITLE
blog post to announce official kick of fundraising

### PR DESCRIPTION
**Pull request by georgedonnelly**

### CODE REVIEW

**Comment by georgedonnelly** [comment]
Foxed the file name, tx

File: _posts/2020-03-05-bitcoin-abc-2020-fundraising.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+We seek your feedback and participation in our sponsorship plan. We stand ready to hear your needs and see how Bitcoin ABC can help you achieve your goals.
+
+The goal of Bitcoin ABC is to build the software infrastructure needed for Bitcoin Cash to be successful. We are “building the roads” for Bitcoin Cash. We are enabling miners to achieve greater efficiency, and therefore greater profits.
+
+Join us in building censorship-resistant P2P electronic cash for the world and realizing the vision that is Bitcoin Cash!
+
+To schedule your presentation of the Bitcoin ABC 2020 Business Plan, please contact George Donnelly, Bitcoin ABC’s business development manager. Please note that priority for presentations is currently assigned to miners, exchanges and other Bitcoin Cash businesses.
+
+george@bitcoinabc.org
+[https://t.me/georgedonnelly](https://t.me/georgedonnelly)
```
Comment (Mengerian) [inline _posts/2020-03-05-bitcoin-abc-2020-fundraising.md]: These last 3 lines need a blank line between them, or else they render on one line.

Comment (georgedonnelly) [reply _posts/2020-03-05-bitcoin-abc-2020-fundraising.md]: Right, will see if I remember how to change that

Comment (georgedonnelly) [reply _posts/2020-03-05-bitcoin-abc-2020-fundraising.md]: blank lines added

**Comment by Mengerian** [comment]
OK, I figured out the problem.

Changing the "2020-03-05" in the filename to "2020-03-04" makes it appear. I guess having a future date in there prevents it from displaying on the website.

So if you could just change that, it should be good to go. (And also in the MultiLangID line inside the file).

File: _posts/2020-03-04-bitcoin-abc-2020-fundraising.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+We seek your feedback and participation in our sponsorship plan. We stand ready to hear your needs and see how Bitcoin ABC can help you achieve your goals.
+
+The goal of Bitcoin ABC is to build the software infrastructure needed for Bitcoin Cash to be successful. We are “building the roads” for Bitcoin Cash. We are enabling miners to achieve greater efficiency, and therefore greater profits.
+
+Join us in building censorship-resistant P2P electronic cash for the world and realizing the vision that is Bitcoin Cash!
+
+To schedule your presentation of the Bitcoin ABC 2020 Business Plan, please contact George Donnelly, Bitcoin ABC’s business development manager. Please note that priority for presentations is currently assigned to miners, exchanges and other Bitcoin Cash businesses.
+
+george@bitcoinabc.org
```
[the remark this answers is not in the document]
Comment (georgedonnelly) [reply _posts/2020-03-04-bitcoin-abc-2020-fundraising.md]: I threw in a span tag

**Pull request merged**